packages/oui-navbar/_variables.less

```diff
@@ -18,13 +18,14 @@
 @oui-navbar-menu-width: rem-calc(300);
 @oui-navbar-menu-padding: rem-calc(4) 0;
 @oui-navbar-menu-background-color: #fff;
-@oui-navbar-menu-background-color_active: @oui-color-foam;
```

Comment:
This will impact the notifications active background from the specs (`.oui-navbar-menu__item`).
To avoid this, we could add variables like:

```
@oui-navbar-menu-link-font-color_active: #fff;
@oui-navbar-menu-link-background-color_active: @oui-color-pacific;
```

Let this one to `@oui-color-pacific`, and change in `navbar.less`:

```
// Menu links
&-menu &-link {
  #oui > .navbar-link_variant(
    @font-color: @oui-navbar-menu-font-color,
  );
  #oui > .navbar-link_menu();

  &:hover,
  &:focus,
  &_active {
    #oui > .navbar-link_variant(
      @background-color: @oui-navbar-menu-link-background-color_active,
      @font-color: @oui-navbar-menu-link-font-color_active
    );
  }
}
```